extensions/hints/init.lua

```diff
@@ -114,7 +149,14 @@ function hints.displayHintsForDict(dict, prefixstring, showTitles)
         else
           local suffixString = ""
           if showTitles then
-            suffixString = ": "..win:title()
+            win_title = win:title()
+            if hints.titleMaxSize > 1 then
+                end_idx = hints.titleMaxSize - 6
+                if end_idx < 6 then end_idx = 6 end
```

Comment:
Note that `end_idx` is based on a static preference. So this does `hints.titleMaxSize - 6 < 6` which is `hints.titleMaxSize < 12`. If the pref is less than twelve always trim to six characters? Why??